satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -1393,7 +1514,7 @@ datasets:
     resolution: 32000
     file_type: nc_fci_asr
     file_key: pixel_percentage
-    long_name: pixel_percentage_used_in_segment
+    standard_name: pixels_used_fraction
```

Comment:
also add `units: %`, set to none in netcdf file.

Reply:
done